scripts/native_arm64.proj

```diff
@@ -0,0 +1,63 @@
+<Project DefaultTargets="Build">
+  <PropertyGroup>
+    <!-- Keep Sdk.targets happy -->
+    <TargetFrameworkIdentifier>net</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>6.0</TargetFrameworkVersion>
+    <UseCompatibilityPackage>true</UseCompatibilityPackage>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="vswhere" IsImplicitlyDefined="true" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <Target Name="Build">
+    <PropertyGroup>
+      <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+      <VSWherePath>$([MSBuild]::NormalizePath('$(Pkgvswhere)', 'tools'))</VSWherePath>
+    </PropertyGroup>
+
+    <!-- Work out VS installation path, so we can find MSBuild.exe -->
+    <Exec
+        Command="vswhere.exe -latest -prerelease -property installationPath -requires Microsoft.Component.MSBuild"
+        WorkingDirectory="$(VSWherePath)"
+        EchoOff="true"
+        ConsoleToMsBuild="true"
+        StandardOutputImportance="Low">
+        <Output TaskParameter="ConsoleOutput" PropertyName="_VSInstallPath" />
+    </Exec>
+
+    <PropertyGroup>
+      <_ArtifactsLogDir>$([MSBuild]::NormalizePath('$(ArtifactsLogDir)', '..', '..', 'log'))</_ArtifactsLogDir>
+      <_OutputPath>$([System.IO.Path]::GetFullPath('$(ArtifactsBinDir)\'))</_OutputPath>
+
+      <_MSBuildCurrentPath>$([MSBuild]::NormalizePath('$(_VSInstallPath)', 'MSBuild', 'Current', 'Bin'))</_MSBuildCurrentPath>
+
+      <_GitExtSshAskPassPath>$([MSBuild]::NormalizePath('$(MSBuildProjectDirectory)', '..', 'GitExtSshAskPass', 'GitExtSshAskPass.sln'))</_GitExtSshAskPassPath>
+      <_GitExtensionsShellExPath>$([MSBuild]::NormalizePath('$(MSBuildProjectDirectory)', '..', 'GitExtensionsShellEx', 'GitExtensionsShellEx.sln'))</_GitExtensionsShellExPath>
+
+      <_ProjectArgsARM64> /p:Configuration=$(Configuration) /p:Platform=ARM64</_ProjectArgsARM64>
+    </PropertyGroup>
+
+    <!-- Build GitExtSshAskPass project, ARM64 -->
+    <Exec
+        Command="msbuild.exe $(_GitExtSshAskPassPath) $(_ProjectArgsARM64) /p:OutDir=$(_OutputPath)GitExtSshAskPass\ /bl:$(_ArtifactsLogDir)\GitExtSshAskPass.binlog"
+        WorkingDirectory="$(_MSBuildCurrentPath)"
+        EchoOff="true"
+        ConsoleToMsBuild="true"
+        StandardOutputImportance="High">
+    </Exec>
+    <!-- Build GitExtensionsShellEx project, ARM64 -->
+    <Exec
+        Command="msbuild.exe $(_GitExtensionsShellExPath) $(_ProjectArgsARM64) /p:OutDir=$(_OutputPath)GitExtensionsShellEx\ /bl:$(_ArtifactsLogDir)\GitExtensionsShellEx64.binlog"
+        WorkingDirectory="$(_MSBuildCurrentPath)"
+        EchoOff="true"
+        ConsoleToMsBuild="true"
+        StandardOutputImportance="High">
+    </Exec>
```

Comment:
Is there a reason we can't do this in native.proj?

Reply:
As I mentioned before, I can't produce the `.msi` installer for WoA, due to the WiX incompatibility problem, so I haven't got a chance to see what's needed for it. I've tried to recompile the WiX library for WoA, but it's so old that it throws so many errors (it's pre-net6 program, and .NET SDK in WoA only supports net6+) that I just give it up.

ConEmu problem in WoA was reported almost 5 years ago, and no progress since. I guess we need to wait a couple of more years still. So for now, I just disable ConEmu in my GE settings in WoA. Actually I want to even hide the `Console` tab so that I don't accidentally click it and produce a zombie process in the background:

![image](https://user-images.githubusercontent.com/49886739/212602329-a1ce5161-c5f3-4b79-9206-4039cda5c4b8.png)

Any idea how to hide it from being displayed on screen?

As to the new `native_arm64.proj` file:
> Is there a reason we can't do this in native.proj?

Nope, just combining them together would cause compile problems on either platforms: they would share objs between different platforms (on WoA) which is a no-no, the x64 compiler wouldn't know how to compile the ARM64 platform which necessitates some needed conditionals to go different compile path for each platform, etc., that I reckon more research are needed to make it work. For now, using a separate proj file for Arm64 is the simplest solution I can think of.

Reply:
> Any idea how to hide it from being displayed on screen?

Look at the 'Build report' tab logic. I think we don't display it always. Maybe it's just a matter of not adding it to the tab collection.

Reply:
> Any idea how to hide it from being displayed on screen?

Make sure these options are always false (and settings disabled) for WoA.

![image](https://user-images.githubusercontent.com/6248932/212630843-02294973-218d-4033-a4fb-b19bd90c20a8.png)

![image](https://user-images.githubusercontent.com/6248932/212631191-d172c0df-be64-4096-bea4-38e560f1c030.png)

Reply:
@gerhardol Yes, the following setting will hide the `Console` tab:

![image](https://user-images.githubusercontent.com/49886739/212804716-5f86d26a-58a0-4857-8ca5-06271a2bda8b.png)